spec/schema.json

```diff
@@ -16,53 +16,346 @@
+        "summary": "Create a new strike-off objection request",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "201": {
+            "description": "An objection request has been created.",
+            "schema": {
+              "$ref": "#/definitions/ResourceCreated"
+            }
+          }
+        }
+      }
+    },
+    "/{objectionId}": {
+      "parameters": [
+        {
+          "name": "objectionId",
+          "in": "path",
+          "required": true,
+          "description": "ID of the Objection",
+          "type": "string"
+        }
+      ],
+      "get": {
+        "summary": "Returns the objection request identified by objectionId",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "a Status object has been returned successfully",
+            "schema": {
+              "$ref": "#/definitions/Objection"
+            }
+          }
+        }
+      },
+      "patch": {
+        "summary": "Updates the strike-off objection identified by objectionId with the values provided",
         "produces": [
           "application/json"
         ],
+        "tags": [
+          "Strike Off Objections"
+        ],
         "parameters": [
           {
             "in": "body",
-            "name": "objection",
-            "description": "Details of the objection",
+            "name": "Patch request",
             "schema": {
-              "$ref": "#/definitions/ObjectionInput"
+              "$ref": "#/definitions/PatchRequest"
             }
           }
         ],
+        "responses": {
+          "204": {
+            "description": "patched successfully"
+          }
+        }
+      }
+    },
+    "/{objectionId}/attachments": {
+      "parameters": [
+        {
+          "name": "objectionId",
+          "in": "path",
+          "required": true,
+          "description": "ID of the Objection",
+          "type": "string"
+        }
+      ],
+      "post": {
+        "summary": "Add an attachment to the Objection",
+        "produces": [
+          "application/json"
+        ],
         "tags": [
           "Strike Off Objections"
         ],
         "responses": {
           "201": {
-            "description": "An objection request has been received.",
+            "description": "An attachment has been added to the Objection",
             "schema": {
-              "$ref": "#/definitions/ObjectionOutput"
+              "$ref": "#/definitions/ResourceCreated"
             }
+          },
+          "400": {
+            "description": "there was an error processing the request"
+          },
+          "404": {
+            "description": "the request or reason does not exist"
```

Comment:
I think this should be 'the objection does not exist'.

Reply:
updated